### ISSUE TITLE
Player hand animation breaks in loop de loop

### ISSUE THREAD

**Issue by isXander** [enjarai/do-a-barrel-roll#64]
**Describe the issue**
When the pitch is wrapped around when going inverted, the player hand suddenly moves slightly to use new pitch values.

**Additional information**
You could probably fix this by modifying the prev pitch value somewhere, I haven't the chance to look.

**Comment by isXander** [comment]
Seems you have done this for yaw in `ElytraMath`

**Comment by carson-coder** [comment]
Is this fixed

**Comment by enjarai** [comment]
Not really, haven't bothered to dig into it too deeply, as its a minor issue.
